core/controllers/topic_editor_test.py

```diff
@@ -66,13 +66,21 @@ def setUp(self) -> None:
         self.save_new_skill(
             self.skill_id_2, self.admin_id, description='Skill Description 2')
         self.topic_id = topic_fetchers.get_new_topic_id()
+        self.topic_id_no_classroom = topic_fetchers.get_new_topic_id()
```

Comment:
Can we rename this variable? Currently, this name is very unclear.

Reply:
yes sure. i guess topic_id_with_no_classroom makes sense